### PR TITLE
Fix Table Refs in Section 2 4

### PR DESCRIPTION
**Pull request by gdfldm**
~I was unable to get table references between sub-documents to work, so I have used a reference to the section instead.~

Fixed table references in section 2.4

### CODE REVIEW

**Comment by blattms** [comment]
I think I messed this one up with one of my merges.
It needs a rebase, sorry.

File: parts/chapters/2.fodt [hunk 1]
```diff
@@ -78973,9 +78973,9 @@ Updated with AFR/TSA Rev-D comments and new keywords.</dc:description><meta:init
...
-    <text:p text:style-name="P449">Finally, see the SCALCERS keyword in the PROPS section, that defines the options used in the re-scaling process, including the options for two point and three point scaling. Whatever option is selected, always verify that the de-normalized curves that will be used in the model are correct and reasonable.</text:p>
-    <text:p text:style-name="P446"/>
+    <text:p text:style-name="P447">As mentioned previously, entering a correct original curve that is not normalized will automatically resolve these type of issues. If a normalized curve is entered, then ensure that simulator can distinguish between SWL and SWCR etc., based on the variable definitions provided in <text:reference-ref text:reference-format="text" text:ref-name="REF_TABLE_OIL_WATER_RELATI__INITIONS_8_2">Table 8.6</text:reference-ref>,<text:s/><text:reference-ref text:reference-format="text" text:ref-name="REF_TABLE_GAS_OIL_RELATIVE__INITIONS_8_2">Table 8.7</text:reference-ref><text:s/>and<text:s/><text:reference-ref text:reference-format="text" text:ref-name="REF_TABLE_GAS_WATER_RELATI__INITIONS_8_2">Table 8.8</text:reference-ref>. </text:p>
```
Comment (blattms) [inline parts/chapters/2.fodt:78976]: On my system there are a lot of page breaks in this area. Do you see that, too?

**Review by blattms — COMMENTED** [review]
References work.
In 2.fodt there seem to be unmotivated page breaks. These were not there before and I don't see how they got introduced.

File: parts/chapters/2.fodt (hunk 1 above)
Comment (gdfldm) [reply parts/chapters/2.fodt:78976]: Yes, looks like I broke the paragraph formatting during the rebase. The new commit should fix this.

**Pull request merged**